tests/api_workflow/test_api_workflow_upload_dataset.py

```diff
@@ -1,22 +1,23 @@
-import copy
 import os
 import pathlib
-import random
 import tempfile
 import warnings
 
 import cv2
 import numpy as np
+import pytest
 import torchvision
 
 from lightly.api.utils import MAXIMUM_FILENAME_LENGTH
 from lightly.data.dataset import LightlyDataset
-from lightly.openapi_generated.swagger_client.models.sample_partial_mode import (
-    SamplePartialMode,
-)
+from lightly.openapi_generated.swagger_client.models import SamplePartialMode
 from tests.api_workflow.mocked_api_workflow_client import MockedApiWorkflowSetup
 
 
+# TODO: fix this text
+@pytest.skip(
+    "Skip this test for now. Test cases need to be updated.", allow_module_level=True
+)
```

Comment:
I believe the test and corresponding code can safely be deleted.

Reply:
Will do that cleanup in a separate PR.